test/routes/actions.test.js

```diff
@@ -0,0 +1,124 @@
+const ApplicationContext = require('../../src/context/application-context');
```

Comment:
I really like the way you performed test on `app` nice job 👍 

I have the feeling you are returning the parameters passed to `app.post` for convenience, but actually, using `jest.fn` you can get the params it has been called with using `myMock.mock.calls[0]` which would return an array of params. 

You might have a reason which forced you to not use this, if so, I would love to know it 😃 

I did not duplicate some of our comment on code having multiple occurence to not pollute the thread unnecessarily

Reply:
> You might have a reason which forced you to not use this, if so, I would love to know it

Hum... no. Thank you for this advice!